src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -110,21 +139,14 @@ public PreparedQuery queryInDatabase(
     return datastore.prepare(new Query(entityName).setFilter(userFilter));
   }
 
-  // TODO: make error handling conform with Richie's error handling
-  // TODO: update to Objectify
+  // TODO: The same GitHub account can be used with multiple emails :(
   /**
-   * Check if a user is stored with userID in Datastore. Invalid return false.
+   * Check if a user is stored with userID in Datastore.
    *
    * @param userID
-   * @return true if userID exists in datastore, false otherwise.
+   * @return PersonEntity with the specified userID or null
    */
-  public Boolean hasProfile(String userID) {
-    Key userKey = KeyFactory.createKey(PersonBuilder.ENTITY_NAME, userID);
-    try {
-      datastore.get(userKey);
-    } catch (EntityNotFoundException e) {
-      return false;
-    }
-    return true;
+  public PersonEntity hasProfile(String userID) {
+    return ofy().load().type(PersonEntity.class).id(userID).now();
   }
```

Comment:
Thanks for leaving this in here for my sake, although I think now it's accessible enough to be done anywhere without this helper function. Also this function isn't static, so it means one would have to instantiate the `UserServlet` class to use this.